<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>searchDeleteWidget</class>
 <widget class="QWidget" name="searchDeleteWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>960</width>
    <height>540</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>微软雅黑 Light</family>
    <pointsize>10</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>搜索与删除信息</string>
  </property>
  <widget class="QTableWidget" name="tableWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>681</width>
     <height>491</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑 Light</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <column>
    <property name="text">
     <string>学号</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>姓名</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>性别</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>生日</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>电话</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>住址</string>
    </property>
   </column>
  </widget>
  <widget class="QPushButton" name="pushButtonSearch">
   <property name="geometry">
    <rect>
     <x>730</x>
     <y>320</y>
     <width>93</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>查询</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>770</x>
     <y>140</y>
     <width>121</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑 Light</family>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>学生信息</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditInfo">
   <property name="geometry">
    <rect>
     <x>732</x>
     <y>260</y>
     <width>201</width>
     <height>31</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>输入相应信息进行搜索</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonDel">
   <property name="geometry">
    <rect>
     <x>840</x>
     <y>320</y>
     <width>93</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>删除</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonCancel">
   <property name="geometry">
    <rect>
     <x>780</x>
     <y>480</y>
     <width>93</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>取消</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioButtonID">
   <property name="geometry">
    <rect>
     <x>730</x>
     <y>210</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑 Light</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>学号</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioButtonName">
   <property name="geometry">
    <rect>
     <x>800</x>
     <y>210</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑 Light</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>姓名</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioButtonTel">
   <property name="geometry">
    <rect>
     <x>870</x>
     <y>210</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑 Light</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>电话</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
